src/main/java/net/sf/jabref/gui/menus/RightClickMenu.java

```diff
@@ -67,6 +69,22 @@ public RightClickMenu(JabRefFrame frame, BasePanel panel) {
         copySpecialMenu.add(new GeneralAction(Actions.COPY_CITE_KEY, Localization.lang("Copy \\cite{BibTeX key}"), KeyBinding.COPY_CITE_BIBTEX_KEY));
         copySpecialMenu.add(new GeneralAction(Actions.COPY_KEY_AND_TITLE, Localization.lang("Copy BibTeX key and title"), KeyBinding.COPY_BIBTEX_KEY_AND_TITLE));
         copySpecialMenu.add(new GeneralAction(Actions.COPY_KEY_AND_LINK, Localization.lang("Copy BibTeX key and link"), KeyBinding.COPY_BIBTEX_KEY_AND_LINK));
+
+        // the submenu will behave dependent on what style is currently selected (citation/preview)
+        PreviewPreferences previewPreferences = Globals.prefs.getPreviewPreferences();
+        String style = previewPreferences.getPreviewCycle().get(previewPreferences.getPreviewCyclePosition());
+        if (CitationStyle.isCitationStyleFile(style)) {
+            JMenu copyCitationMenu = new JMenu(Localization.menuTitle("Copy citation..."));
+            copyCitationMenu.add(new GeneralAction(Actions.COPY_CITATION_HTML, "HTML", KeyBinding.COPY_PREVIEW));
```

Comment:
### Preview is selected
![preview](https://cloud.githubusercontent.com/assets/15333371/21139041/42c8e7fa-c131-11e6-954a-a71e7a09ed4b.png)

### Citation style is selected
![citation](https://cloud.githubusercontent.com/assets/15333371/21139042/42d55d50-c131-11e6-87d7-fb05a57bc5a8.png)